tests/package.json

```diff
@@ -4,16 +4,16 @@
     "dev": "node server.js",
     "clean": "rm -rf .next static/manifest",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
+    "start": "yarn clean && yarn build && NODE_ENV=production node server.js",
     "test": "yarn clean && jest",
     "export": "yarn build && next export"
   },
   "dependencies": {
     "mkdirp": "^0.5.1",
     "next": "7.0.2",
     "next-compose-plugins": "^2.1.1",
-    "next-manifest": "^1.6.0",
-    "next-offline": "latest",
+    "next-manifest": "^2.0.0",
```

Comment:
This wasn't strictly necessary, but since I was messing with all this stuff, I wanted to make sure nothing was broken here.

Reply:
eh, this is probably ok.

If we can actually entirely remove `next-manifest` it might be nice.

I added it mainly to test compatibility w/ `next-compose-plugins`